static_file_handler.cc

```diff
@@ -56,16 +62,30 @@ StaticFileHandler::Status StaticFileHandler::HandleRequest(const Request& reques
     if (ReadFile(file_path, file_content)) {
         response->SetStatus(Response::ResponseCode::OK);
 
-        std::string content_type = GetMimeType(file_path);
+        bool isMarkdown = false;
+        std::string content_type = GetMimeType(file_path, isMarkdown);
```

Comment:
Hmm, now that I think about it, checking if a file is a .md file is simpler than that. Will go ahead and make this change